plugins/woocommerce/templates/cart/cart.php

```diff
@@ -74,6 +74,19 @@
 
 						<td class="product-thumbnail">
 						<?php
+						/**
+						 * Filter the product thumbnail displayed in the WooCommerce cart.
+						 *
+						 * This filter allows developers to customize the HTML output of the product
+						 * thumbnail. It passes the product image along with cart item data
+						 * for potential modifications before being displayed in the cart.
+						 *
+						 * @param string $thumbnail     The HTML for the product image.
+						 * @param array  $cart_item     The cart item data.
+						 * @param string $cart_item_key Unique key for the cart item.
+						 *
+						 * @since 2.1.0
+						 */
```

Comment:
I'm fine with keeping this according to the Boy Scout rule and because documentation is always great, but the latest changes make this look like an addition unrelated to the current PR.

Reply:
Yes, you are completely right, but I couldn't bring myself to delete this DocBlock. 😄